packages/sources/coingecko/src/endpoint/crypto.ts

```diff
@@ -101,20 +101,43 @@ const handleBatchedRequest = (
 }
 
 export const execute: ExecuteWithConfig<Config> = async (request, context, config) => {
-  const validator = new Validator(request, inputParameters, {}, { overrides })
+  const validator = new Validator(request, inputParameters, {}, { overrides: internalOverrides })
```

Comment:
If we are moving toward using the Overrider do we still need to pass `overrides` to the Validator?

Reply:
The validator would still throw an error if the format is invalid, but I also to a check on the validity in the Overrider too so no, it is not necessary